Description: Fix autopkgtest with zsh-static due to (correct) warning on STDERR
Author: Jun T <takimoto-j@kba.biglobe.ne.jp>
Origin: https://zsh.org/workers/50052

diff --git a/Test/V14system.ztst b/Test/V14system.ztst
index 100daab08..ffdb730a4 100644
--- a/Test/V14system.ztst
+++ b/Test/V14system.ztst
@@ -5,10 +5,10 @@
   if zmodload -s zsh/system && zmodload -s zsh/zselect; then
     tst_dir=V14.tmp
     mkdir -p -- $tst_dir
+    : > $tst_dir/file # File on which to acquire flock.
   else
     ZTST_unimplemented='the zsh/system and zsh/zselect modules are not available'
   fi
-  : > $tst_dir/file # File on which to acquire flock.
 
 %test
 
